projects/sonar/src/manual_translations.ts

```diff
@@ -178,6 +178,10 @@ _('contribution_role_cre');
 _('contribution_role_prt');
 _('contribution_role_dgs');
 
+// Investigator role
+_('investigator');
```

Comment:
Are not theses values in the JSONSchema?

Reply:
Yes they are, but I'm not using the translations from backend in UI yet..